src/libraries/System.Security.Permissions/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,1133 @@
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Data.Odbc.OdbcPermissionAttribute</Target>
+    <Left>lib/netstandard2.0/System.Security.Permissions.dll</Left>
+    <Right>lib/net461/System.Security.Permissions.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Data.Common.DBDataPermission</Target>
+    <Left>lib/netstandard2.0/System.Security.Permissions.dll</Left>
+    <Right>lib/net461/System.Security.Permissions.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Data.Common.DBDataPermissionAttribute</Target>
+    <Left>lib/netstandard2.0/System.Security.Permissions.dll</Left>
+    <Right>lib/net461/System.Security.Permissions.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Configuration.ConfigurationPermission</Target>
+    <Left>lib/netstandard2.0/System.Security.Permissions.dll</Left>
+    <Right>lib/net461/System.Security.Permissions.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Configuration.ConfigurationPermissionAttribute</Target>
+    <Left>lib/netstandard2.0/System.Security.Permissions.dll</Left>
+    <Right>lib/net461/System.Security.Permissions.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Security.IStackWalk</Target>
+    <Left>lib/netstandard2.0/System.Security.Permissions.dll</Left>
+    <Right>lib/netcoreapp3.1/System.Security.Permissions.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Security.PermissionSet</Target>
+    <Left>lib/netstandard2.0/System.Security.Permissions.dll</Left>
+    <Right>lib/netcoreapp3.1/System.Security.Permissions.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Security.Permissions.PermissionState</Target>
+    <Left>lib/netstandard2.0/System.Security.Permissions.dll</Left>
+    <Right>lib/netcoreapp3.1/System.Security.Permissions.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Xaml.Permissions.XamlLoadPermission</Target>
+    <Left>ref/netcoreapp3.0/System.Security.Permissions.dll</Left>
+    <Right>lib/netstandard2.0/System.Security.Permissions.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Xaml.Permissions.XamlLoadPermission</Target>
+    <Left>lib/netcoreapp3.0/System.Security.Permissions.dll</Left>
+    <Right>lib/netstandard2.0/System.Security.Permissions.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0008</DiagnosticId>
```

Comment:
This is different, but another one related to missing dependencies.  NamedPermissionSet derives from PermissionSet which is part of System.Runtime.  This might be interesting to look at @safern.  One thing I notice is that the suppression doesn't list the interface that was removed, which means it will suppress all removals.  Another thing that surprised me was that we should be able to identify that the interface is still present with the exact same name, we just couldn't be certain if it would resolve to the same definition as the one defined in netstandard.dll.  I wonder if we want to tolerate that, or give it a different diagnostic?

Reply:
Do you want to track this discussion in an issue so that it doesn't get lost?